__tests__/api.test.js

```diff
@@ -89,7 +89,62 @@ describe('GET /api/articles/:article_id', () => {
         });
     })
 })
-
+describe('POST /api/articles/:article_id/comments', () => {
+    test("returns the new comment that has been created along with 201 status", () => {
+        return request(app)
+        .post("/api/articles/2/comments")
+        .send({
+            username: "butter_bridge",
+            body: "well well well"
+        })
+        .expect(201)
+        .then(({body}) => {
+            const comment = body.comment[0]; 
```

Comment:
Once the model is fixed (see my other comment) comment will be an object, not an array.

Reply:
done